dist/auth-k8s/index.js

```diff
@@ -4182,13 +4178,16 @@ function writeConfiguration(config) {
         const tempDir = yield io.makeTempDirectory();
         const configPath = path.join(tempDir, 'bot-config.yaml');
         const data = yaml.stringify(config);
+        core.debug('Writing tbot configuration to ' + configPath);
+        core.debug('Configuration value:\n' + data);
```

Comment:
Do we want to print the data itself?

Reply:
Yeah - there's nothing sensitive here and I imagine it could be helpful for debugging. These logs won't show unless their CI run fails and they re-run it in debug mode.

Reply:
Didn't realize that. Makes sense